app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+{/\* \* Copyright © 2023 United States Government as represented by the
+
+- Administrator of the National Aeronautics and Space Administration.
+- All Rights Reserved.
+-
+- SPDX-License-Identifier: Apache-2.0 \*/}
+
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardGroup,
+  CardHeader,
+  CardMedia,
+  GridContainer,
+  Icon,
+  Link,
+  Tag,
+} from '@trussworks/react-uswds'
+
+import ParallaxImage from './labs/parallax/ParallaxImage'
+
+import React from 'react'
+
+import styles from './labs/across.css'
+import mma from './labs/mma.png'
+
+  <div style={{ backgroundColor: '#0c0f18', color: 'white', minHeight: '2000px' }}>
+
+    <ParallaxImage
+      src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg"
+      alt="BNS"
+    />
+
+    <GridContainer className="usa-section" style={{minWidth:'75%'}}>
+
+      <div className="section-head"><h1>Overview</h1></div>
```

Comment:
I don't see the `section-head` class defined anywhere.

Reply:
Indeed, these were defined in the across.css file, but never actually used, so I've removed the class reference. In any case, these section headers have now been replaced by Grid rows.